docs/rules/no-duplicate-imports.md

```diff
@@ -12,59 +14,51 @@ import { find } from 'module';
 
 ## Rule Details
 
+This rules requires that all imports from a single module exists in a single `import` statement.
+
 This inspection reports any duplicated module in an import statement.
```

Comment:
Sorry my original line comment was unclear. Two choices about what is now the second sentence:
* If it is no longer needed, delete it.
* If it is still needed, s/inspection/rule/ and add backticks to `import`